### DISCUSSION TITLE
[Feature] UI Indicator for RAW images

### DISCUSSION THREAD

**Discussion by nathanrdodson** [immich-app/immich#8027] [Feature Request]
### The feature

Hi! Inevitably, I would love to use Immich for all of my photography and image management needs. Currently, I use Apple Photos for managing my photos (I have an iPhone and a Mac, but my desktop PC is Windows and I dual boot Linux - a cross-platform solution would be game changing :D).

As a hobby photographer and a data-hoarder, management of RAW photos is a necessary part of my workflow. One thing I love about Apple Photos is that it marks RAW images with a small badge in all of the gallery UIs, like this:

<img width="1131" alt="Screenshot 2024-03-17 at 1 20 15 PM" src="https://github.com/immich-app/immich/assets/32574295/3b3f6e22-1a40-44c7-81c0-5f84c4231dfe">

I think implementing this in Immich would be really helpful - additionally, having a way to filter RAW photos in the timeline if desired (either filter them in or out) would be really helpful, too.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by mertalev** [comment]
I think this would be nice. We'd need to add a DB column for whether an asset is raw and have a list of raw format extensions to check when importing an asset.

In terms of what you can do right now, it's possible to search for assets of a certain raw format using filename search, e.g. searching `.dng`. The reverse isn't supported right now.

**Comment by bo0tzz** [comment]
#21029

**Reply by antoniovj1** [reply]
Just added a related comment https://github.com/immich-app/immich/discussions/21029#discussioncomment-15149098
